doc/news/_preparation_next_release.md

```diff
@@ -129,6 +129,7 @@ _(Michael Tucek)_
 
 - Really add all tools when using `-DTOOLS=ALL`. _(Markus Raab)_
 - ZeroMQ Hub: fix compilation and man page. _(Markus Raab)_
+- ambiguous write operations now disabled in kdb _(Alexander Firbas)_
```

Comment:
Please give more details here (e.g. that it is about cascading keys, also link to the tutorial.), you can even make a highlight out of this.

Reply:
I have updated the release notes with further details.